src/main/java/uk/gov/pay/api/model/directdebit/search/DDSearchResponse.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.pay.api.model.directdebit.search;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.pay.api.model.IPaymentSearchPagination;
+import uk.gov.pay.api.model.links.PaymentSearchNavigationLinks;
+
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class DDSearchResponse implements IPaymentSearchPagination {
```

Comment:
Not sure about `DD` prefix as name, we usually use `DirectDebit`. I understand that it may be too long, so I don't mind, if you would like please check with someone else as well

Reply:
It's under package `directdebit` so it should be self explanatory why we use DD (to avoid mile long class names)